examples/v1beta1/triggertemplates/triggertemplate.yaml

```diff
@@ -14,6 +14,7 @@ spec:
     default: This is the default message
   - name: contenttype
     description: The Content-Type of the event
+    default: text/plain
```

Comment:
In this demo, without a default value and without assigning value in `TriggerBinding`, the default value `$(tt.params.contenttype)` will be retained in `TaskRun`, which is quite strange.

Reply:
Do you mean to say we get `$(tt.params.contenttype)` when there's no value in TriggerBinding and no default value? It should during processing of events then,

Reply:
Yes, I understand that this value should be declared in the `TriggerBinding` as the source of the `contenttype` variable, which can be either `$(body.xxx)` or `$(extensions.xxx)`. However, I am not aware of any existing data source to obtain this parameter value, and it is not mentioned in the documentation.

So my current solution is to give it a default value first, which will not affect other `TriggerBindings` assigning it a suitable value in the future.

Reply:
I referred to the code here and also added a configuration in `TriggerBinding` to assign a value to `contenttype`.

https://github.com/tektoncd/triggers/blob/87e123e173028bd2d69547386b09ffda996a0394/examples/v1beta1/trigger-ref/triggerbinding.yaml#L5-L12

Reply:
If there is no default value, it will not necessarily cause a problem.
It will only cause an issue if there is also no value assigned in the TriggerBinding.